qt/scientific_interfaces/ISISReflectometry/GUI/Options/OptionsDialogPresenter.cpp

```diff
@@ -0,0 +1,87 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI,
+//     NScD Oak Ridge National Laboratory, European Spallation Source
+//     & Institut Laue - Langevin
+// SPDX - License - Identifier: GPL - 3.0 +
+
+#include "OptionsDialogPresenter.h"
+#include "IOptionsDialogView.h"
+
+namespace MantidQt {
+namespace CustomInterfaces {
+namespace ISISReflectometry {
+
+/**
+ * Construct a new presenter with the given view
+ * @param view :: a handle to a view for this presenter
+ * @param model :: a handle to a model for this presenter
+ */
+OptionsDialogPresenter::OptionsDialogPresenter(
+    IOptionsDialogView *view, std::unique_ptr<IOptionsDialogModel> model)
+    : m_view(view), m_model(std::move(model)), m_mainWindowNotifyee() {}
+
+/** Allow options to be initialised once the main presenter
+    is constructed (necessary due to rounding impl)
+ */
+void OptionsDialogPresenter::notifyInitOptions() { initOptions(); }
```

Comment:
I can't quite remember at the moment, so I will make the change but it might be that we have to change it back once I merge these changes into #27660